python/cudf/dataframe/column.py

```diff
@@ -602,5 +603,29 @@ def _pointer(self):
         """
         return column_view_pointer(self)
 
+    @property
+    def is_unique(self):
+        return self.unique_count() == len(self)
+
+    @property
+    def is_monotonic(self):
+        return self.is_monotonic_increasing
+
+    @property
+    def is_monotonic_increasing(self):
+        if not hasattr(self, '_is_monotonic_increasing'):
+            self._is_monotonic_increasing = (
+                series.Series(self)[1:] >= series.Series(self)[:-1]
+            ).all()
```

Comment:
We shouldn't need to construct a Series in order to do this here. You should be able to use the binop functionality to do this.

Reply:
This will actually not do what you want as Slicing the Series will misalign the indexes and then the binaryop will realign them as well.

Reply:
Not sure I understand your second comment.  However, I added `numeric_column_binop`-based methods to `NumericalColumn` (so the unused `Column` methods could be removed) - Is this reasonable?

Reply:
calling `series.Series(self)` will construct a Series with a index of 0 --> N (where N is the length of the Series), then slicing it with `1:` will make the make the Index 1 --> N, while slicing it with `:-1` will make the index 0 --> N-1. When doing the `>=` binaryop the first thing it will do is align the two Series based on their indexes which is not what you want to do as everything will always return `True`.